File: VB/CustomizeExportActionVB.Win/WinApplication.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports DevExpress.ExpressApp.Win

Partial Public Class CustomizeExportActionVBWindowsFormsApplication
	Inherits WinApplication
	Public Sub New()
		InitializeComponent()
        DelayedViewItemsInitialization = True
	End Sub

	Private Sub CustomizeExportActionVBWindowsFormsApplication_DatabaseVersionMismatch(ByVal sender As Object, ByVal e As DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs) Handles MyBase.DatabaseVersionMismatch

        e.Updater.Update()
        e.Handled = True

    End Sub
End Class

File: CS/CustomizeExportAction.Win/WinApplication.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using DevExpress.ExpressApp.Win;
using DevExpress.ExpressApp.Updating;
using DevExpress.ExpressApp;

namespace CustomizeExportAction.Win {
    public partial class CustomizeExportActionWindowsFormsApplication : WinApplication {
        public CustomizeExportActionWindowsFormsApplication() {
            InitializeComponent();
        }

        private void CustomizeExportActionWindowsFormsApplication_DatabaseVersionMismatch(object sender, DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs e) {
			e.Updater.Update();
			e.Handled = true;
        }
    }
}
